js/web/script/test-runner-cli-args.ts

```diff
@@ -366,10 +366,7 @@ export function parseTestRunnerCliArgs(cmdlineArgs: string[]): TestRunnerCliArgs
   //       we need this for now because Chrome does not support webnn yet,
   //       and ChromeCanary is not in CI.
 
-  // TODO: web CI is still using chrome v112, where WebGPU is not available yet.
-  // re-enable webgpu after CI upgraded chrome to v113.
-  // const defaultBrowserBackends = ['webgl', 'webgpu', 'wasm', 'xnnpack' /*, 'webnn'*/];
-  const defaultBrowserBackends = ['webgl' /*, 'webgpu' */, 'wasm', 'xnnpack' /*, 'webnn'*/];
+  const defaultBrowserBackends = ['webgl', 'webgpu', 'wasm', 'xnnpack' /*, 'webnn'*/];
```

Comment:
wonder if we should remove xnnpack

Reply:
I think we have a few operators implemented in xnnpack (conv for example) and our default build use `--use_xnnpack`. if we want this part to be tested, we can either:
- keep the current list, or
- remove xnnpack from defaultBrowserBackends , but run `npm test -- suite1 -b=xnnpack` separately.